staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/oidc/oidc.go

```diff
@@ -338,6 +351,27 @@ func New(opts Options) (*Authenticator, error) {
 	return authenticator, nil
 }
 
+// roundTripper is a http.RoundTripper that rewrites the issuer URL to the
+// discovery URL for the /.well-known/openid-configuration endpoint.
+type roundTripper struct {
```

Comment:
Give this struct a more specific name.

Reply:
sounds good! I was thinking I'll use the same one for the jwks metrics and hence the generic name, but I can just have different roundTrippers each scoped to a specific task and layered?